packages/round-manager/src/features/round/__tests__/ViewApplicationPage.test.tsx

```diff
@@ -219,6 +234,53 @@ describe("ViewApplicationPage", () => {
       });
     });
   });
+
+  it("shows error modal when reviewing application fails", async () => {
+    renderWithContext(<ViewApplicationPage />, {
+      IPFSCurrentStatus: ProgressStatus.IS_ERROR,
+    });
+
+    expect(await screen.findByTestId("error-modal")).toBeInTheDocument();
+  });
+
+  it("choosing done closes the error modal", async () => {
+    renderWithContext(<ViewApplicationPage />, {
+      IPFSCurrentStatus: ProgressStatus.IS_ERROR,
```

Comment:
^ not entirely sure how to trigger and error state here to see if the modal pops up